tools/doc/markdown.js

```diff
@@ -10,10 +10,18 @@ function replaceLinks({ filename, linksMapper }) {
   return (tree) => {
     const fileHtmlUrls = linksMapper[filename];
 
+    visit(tree, (node) => {
+      if (node.url) {
+        node.url = node.url.replace(
+          /^([^/]+)\.md(#.+)?$/,
```

Comment:
@aduh95, do you think it possible to have this regular expression unit tested? I imagine it would be located in the same place as the last one you did.  If it doesn't yet exist, I'm sure it would make @richardlau happy to see one. I should be able to be more helpful in a couple (2) days. Thanks for your patience.

Reply:
Is it done somewhere else in the code base? I could use a little help from an example.

Reply:
OK, please have a look to 3c37b95992c4ec4303aab016c3d60ddd70be77b5.

/cc @DerekNonGeneric @richardlau